# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/default-term.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/src/config/defaultConfig.hpp.vanilla	2024-11-23 14:30:19.433279957 +0100
+++ hyprland-source/src/config/defaultConfig.hpp	2024-11-23 14:31:57.139945039 +0100
@@ -9,7 +9,7 @@
 # OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
 # #######################################################################################
 
-autogenerated = 1 # remove this line to remove the warning
+autogenerated = 0 # remove this line to remove the warning
 
 # This is an example Hyprland config file.
 # Refer to the wiki for more information.
@@ -38,7 +38,7 @@
 # See https://wiki.hyprland.org/Configuring/Keywords/
 
 # Set programs that you use
-$terminal = kitty
+$terminal = alacritty
 $fileManager = dolphin
 $menu = wofi --show drun
 
